src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,12 +6,14 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
+        public static int RandomSeed { get; } = Guid.NewGuid().GetHashCode();
```

Comment:
@benaadams I did this in response to @jkotas' suggestion at https://github.com/dotnet/corefx/issues/14046#issuecomment-263295578. I don't think it is worth it to be too prudent randomizing this; if someone is smart enough, they can figure it out anyway. e.g.

```cs
int GetTheRandomSeed()
{
    int tuple = (1, 1);
    return Combine(1, Combine(1, 0)) ^ tuple.GetHashCode();
}
```